jetty-integrations/jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -123,15 +290,47 @@ public OpenIdConfiguration(@Name("issuer") String issuer,
                                @Name("clientSecret") String clientSecret,
                                @Name("authenticationMethod") String authenticationMethod,
                                @Name("httpClient") HttpClient httpClient)
+    {
+        this(issuer, clientId, clientSecret, authorizationEndpoint, tokenEndpoint, endSessionEndpoint, authenticationMethod, httpClient, false, false, Collections.emptyList());
+    }
+
+    /**
+     * Create an OpenID configuration for a specific OIDC provider.
+     * @param issuer The URL of the OpenID provider.
+     * @param clientId OAuth 2.0 Client Identifier valid at the Authorization Server.
+     * @param clientSecret The client secret known only by the Client and the Authorization Server.
+     * @param authorizationEndpoint the URL of the OpenID provider's authorization endpoint if configured.
+     * @param tokenEndpoint the URL of the OpenID provider's token endpoint if configured.
+     * @param endSessionEndpoint the URL of the OpdnID provider's end session endpoint if configured.
+     * @param authenticationMethod Authentication method to use with the Token Endpoint.
+     * @param httpClient The {@link HttpClient} instance to use.
+     * @param authenticateNewUsers Whether to authenticate new users.
+     * @param logoutWhenIdTokenIsExpired Whether to logout when the ID token is expired.
+     * @param scopes The scopes to request.
+     */
+    private OpenIdConfiguration(@Name("issuer") String issuer,
+                                @Name("clientId") String clientId,
+                                @Name("clientSecret") String clientSecret,
+                                @Name("authorizationEndpoint") String authorizationEndpoint,
+                                @Name("tokenEndpoint") String tokenEndpoint,
+                                @Name("endSessionEndpoint") String endSessionEndpoint,
+                                @Name("authenticationMethod") String authenticationMethod,
+                                @Name("httpClient") HttpClient httpClient,
+                                @Name("authenticateNewUsers") boolean authenticateNewUsers,
+                                @Name("logoutWhenIdTokenIsExpired") boolean logoutWhenIdTokenIsExpired,
+                                @Name("scopes") List<String> scopes)
```

Comment:
You don't need the @Name annotation on a private constructor as it will never be used in XML